code/modules/cargo/exports/organs.dm

```diff
@@ -1,6 +1,13 @@
 /datum/export/organ
 	include_subtypes = FALSE //CentCom doesn't need organs from non-humans.
 
+/datum/export/organ/get_cost(obj/exported_item, apply_elastic)
+	if(HAS_TRAIT(exported_item, TRAIT_CLIENT_STARTING_ORGAN))
+		// 10x export mult for organs that started in a player
+		// Unaffected by price elasticity as there's a limited amount of these in play
+		return round(init_cost * 10)
```

Comment:
Hit this with a define to undef at the bottom of the file to avoid a magic number

Reply:
o7 on it